Makefile

```diff
@@ -34,39 +32,45 @@ docker:
 	docker cp hugo-build:/go/bin/hugo .
 	docker rm hugo-build
 
+govendor:
+	go get -u github.com/kardianos/govendor
+	go install github.com/kardianos/govendor
```

Comment:
You're right.  I was looking at this on my phone and misunderstood where your comment was in the Makefile.  The `go install` does indeed need to be removed.